src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -65,7 +66,17 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
   const notifications = useSelector(({ chrome: { notifications } }: ReduxState) => notifications?.data || []);
   const isOrgAdmin = useSelector(({ chrome }: ReduxState) => chrome.user?.identity.user?.is_org_admin);
 
+  async function getNotifications() {
+    try {
+      const notifications = await axios.get('/api/notifications/v1/notifications/drawer');
+      dispatch(populateNotifications(notifications.data.data));
+    } catch (error) {
+      console.error('Unable to get Notifications ', error);
+    }
+  }
+
   useEffect(() => {
+    getNotifications();
```

Comment:
The drawer should be populated on the Chrome start-up. Even if the drawer is closed. We need the data to change the toggle state if needed.

Getting the data after the drawer is opened is too late.